cranelift/codegen/src/verifier/mod.rs

```diff
@@ -1499,16 +1499,6 @@ impl<'a> Verifier<'a> {
                                 ),
                             ));
                         }
-                        if arg_type.lane_bits() >= ctrl_type.lane_bits() {
```

Comment:
Very nice to get rid of the `lane_bits` checks in the verifier. Can't we also get rid of the `lane_count` checks? And at that point delete the `typecheck_special` cases for the `Unary` format entirely?

Reply:
Hah, the only remaining checks were for vector arguments, and none of those instructions support vectors now. Great catch!